tests/factories.py

```diff
@@ -225,7 +225,7 @@ class Meta:
 
 
 class TextFieldFactory(FieldFactory):
-    settings = {"type": FieldType.text.value}
+    settings = {"type": FieldType.text.value, "use_markdown": False}
```

Comment:
I would say to keep using the default values for `use_markdown` and use the desired values in tests